ES/ESdd007.xml

```diff
@@ -365,109 +394,90 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                              <ab type="pricking">Primary pricks are visible.</ab>
+                              <ab type="pricking">Ruling pricks are visible.</ab>
+                              <ab type="ruling">The upper line is written above the ruling.</ab>
+                              <ab type="ruling">The bottom line is written above the ruling.</ab>
                            </layout>
                         
                            <layout/>
                         </layoutDesc>
                      </objectDesc>
                      <handDesc>
                      
-                        <handNote xml:id="h1" script="Ethiopic">Hand a (Text I): <locus from="2r" to="286v">2r-286v</locus>
-                           <seg type="script">careful, by a well-trained hand; Hand b: careful,
-                           neat, by a well-trained hand</seg>
-                           <seg type="ink">Black; red</seg>
-                           <note>Ethiopic number “6” looks like
-                           “compressed” number “7” (<locus target="#42r">42r</locus>, 123r, 202r,
-                           286va).</note>
-                        The scribe
-                        is <persName role="scribe" ref="PRS00000">Malkǝʾa Krǝstos</persName> (s. the colophon).
-                        <date when="1610">1610</date>
-                           <seg type="rubrication">Nomina sacra; names of the King, Metropolitan,
-                           commissioner and scribe of Text I; 1st ,
-                           3rd , 5th lines of the incipit page of Text I); one or two initial lines
-                           of sub-sections of Text I; initial words referring to parts of Text I
-                           (e.g., fol. 253rb); 
-                           elements of the punctuation signs; entire Ethiopic numerals or their
-                           elements; elements of quire marks.</seg>
+                        <handNote xml:id="h1" script="Ethiopic">
+                           <locus from="2r" to="286v"/>
+                           <seg type="script">Seventeenth-century script.</seg>
+                           <seg type="ink">Black, red.</seg>
+                           <desc>Careful, by a well-trained hand; Ethiopic number ‘6’ looks like ‘compressed’ number ‘7’ (<locus target="#42r #23r #202r #286va"/>).
+                           The scribe is <persName ref="PRS14616MalkeaKrestos" role="scribe">Malkǝʾa Krǝstos</persName>, see <ref target="#coloph1">the colophon</ref>, who completed the manuscript in
+                              <date when="1610">1610</date>. </desc>
+                           
+                           <seg type="rubrication">Divine names; names of the <persName ref="PRS9038Susenyos">king</persName>, <persName ref="PRS8740Semon">metropolitan</persName>,
```

Comment:
I think `<persName ref="PRS8740Semon">metropolitan</persName>` might not be visualised well, maybe "`metropolitan <persName ref="PRS8740Semon"/>`" would be better?